core/models.py

```diff
@@ -560,8 +560,8 @@ class HistoryModel(DirtyFieldsMixin, models.Model):
     json_ext = FallbackJSONField(db_column="Json_ext", blank=True, null=True)
     date_created = models.DateTimeField(db_column="DateCreated", null=True)
     date_updated = models.DateTimeField(db_column="DateUpdated", null=True)
-    user_created = models.ForeignKey(User, db_column="UserCreatedUUID", related_name='user_created', on_delete=models.deletion.DO_NOTHING)
-    user_updated = models.ForeignKey(User, db_column="UserUpdatedUUID", related_name='user_updated', on_delete=models.deletion.DO_NOTHING)
+    user_created = models.ForeignKey(User, db_column="UserCreatedUUID", related_name='%(class)s_user_created', on_delete=models.deletion.DO_NOTHING, null=True)
```

Comment:
I think this must be mandatory. @delcroip can you confirm this?

Reply:
According to the https://openimis.atlassian.net/wiki/spaces/OP/pages/1904738367/Update+of+the+core+module - this field is 'technical'.  ("Technical fields have **")
Moreover the user who creates object is assigned automatically as the value of this field ('user_created') after saving object. And on every update the value of "user_updated" is changed to the user, who updated this object. This field is updated automatically on every save/update or deletion/replacement. Also 'user_created' is updated automatically after saving newly created object.

Reply:
yes it is force by backend so why not forbinding null ?

Reply:
Ok @delcroip and @dragos-dobre - I've changed that and it works after changing it.
Ok, I have to make changes on another repo "policy holder backend" after Patrick's review.